### PR TITLE
Remove `if VERSION ≥ v"1.4"` etc.

### PR DESCRIPTION
**Pull request by mcabbott**
since master only supports 1.6+

### CODE REVIEW

File: test/DualTest.jl [hunk 1]
```diff
@@ -589,7 +585,7 @@ ForwardDiff.:≺(::Type{OuterTestTag}, ::Type{TestTag}) = false
         @test pq isa Tuple{Dual{TestTag()},Dual{TestTag()}}
         # We have to adjust tolerances if lower accuracy is requested
         # Therefore we don't use `dual_isapprox`
-        tol = V === Float32 ? 5f-4 : 1e-6
+        tol = V === Float32 ? 5f-4 : 1e-5
```
Comment (devmotion) [inline test/DualTest.jl:588]: This seems unrelated?

**Comment by devmotion** [comment]
Apart from the question above the PR seems fine :slightly_smiling_face:

File: test/DualTest.jl (hunk 1 above)
Comment (mcabbott) [reply test/DualTest.jl:588]: I think this is an intermittent failure on 1.8, e.g. here:

https://github.com/JuliaDiff/ForwardDiff.jl/actions/runs/3601997063/jobs/6068433780#step:6:445
